packages/manager/modules/pci/src/projects/project/analytics-data-platform/analytics-data-platform.html

```diff
@@ -0,0 +1,56 @@
+<div data-ui-view>
+    <oui-page-header 
+        class="mb-2" 
+        data-heading="{{ ::'analytics_data_platform_title' | translate }}">
+    </oui-page-header>
+    <div>
+        <!-- error messages -->
+        <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+        <oui-datagrid class="mt-3" data-rows="$ctrl.clusters">
+            <oui-column data-title="::'analytics_data_platform_common_cluster_name' | translate"
+                        data-property="clusterName"
+                        data-sortable="asc"
+                        data-type="string"
+                        data-searchable data-filterable>
+                        <oui-button 
+                            data-variant="link"
+                            data-on-click="$ctrl.manageCluster($row.serviceName)">
+                            <span data-ng-bind="::$row.clusterName"></span>
+                        </oui-button>
+            </oui-column>
+            <oui-column data-title="::'analytics_data_platform_common_status' | translate"
+                        data-property="status"
+                        data-type="string"
+                        data-sortable data-searchable data-filterable>
+                <span class="oui-status"
+                        data-ng-class="'oui-status_' + $ctrl.STATUS_CLASS[$row.status]"
+                        data-ng-bind=":: ('analytics_data_platform_common_status_' + $row.status) | translate">
+                </span>
+            </oui-column>
+            <oui-column data-title="::'analytics_data_platform_common_region' | translate"
+                        data-property="osRegion" data-sortable>
+                        <span data-ng-bind="::$ctrl.cucRegionService.getTranslatedMicroRegion($row.osRegion)"></span>
+            </oui-column>
+            <oui-column data-title="::'analytics_data_platform_common_version' | translate"
+                        data-property="clusterType"
+                        data-sortable></oui-column>
+            <oui-action-menu data-align="end" data-compact>
+                <oui-action-menu-item 
+                    data-disabled="$row.status === $ctrl.ADP_STATUS.IN_PROGRESS"
+                    data-on-click="$ctrl.manageCluster($row.serviceName)">
+                    <span data-translate="analytics_data_platform_list_table_button_manage_cluster"></span>
+                </oui-action-menu-item>
+            </oui-action-menu>
+            <extra-top>
+                <oui-button data-on-click="$ctrl.deployCluster()">
+                    <span data-translate="analytics_data_platform_list_create_button_label"></span>
+                </oui-button>
+                <button class="float-right oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
```

Comment:
Don't use `float-right`, use boostrap `row` and columns

Reply:
This is done